src/Ladybug_Create LB Header.py

```diff
@@ -92,13 +101,24 @@ def setDefaults():
     
     return checkData, location, dataType, units, timeStep, analysisPeriod
 
+def checkRawData(rawD):
+    lb_preparation = sc.sticky["ladybug_Preparation"]()
+    indexList, oldHeader = lb_preparation.separateList(rawD, lb_preparation.strToBeFound)
+    separatedLists = rawD[indexList[0]+7:indexList[1]]
```

Comment:
@mostaphaRoudsari  it will add a default header that CreateHeader originally generates